pipeline-steps/aio_prepare.groovy

```diff
@@ -0,0 +1,39 @@
+def prepare(){
+  common.conditionalStage(
+    stage_name: "Prepare Deployment",
+    stage: {
+      dir("/opt/rpc-openstack"){
+        git branch: env.RPC_BRANCH, url: env.RPC_REPO
+        sh "git submodule update --init"
+        withEnv([
+          "DEPLOY_AIO=yes",
+          "DEPLOY_OA=no",
+          "DEPLOY_RPC=no"
```

Comment:
Do you want OSA to be deployed at this stage?  I think `DEPLOY_OA` defaults to `yes`.